### ISSUE TITLE
Issues regarding the metrics of comma

### ISSUE THREAD

**Issue by gorjious** [googlefonts/googlesans#54]
In trying to determine the kerning for combinations which include Armenian letters with Latin punctuation, I noticed some inconsistencies in the sidebearings of the comma. 

The main concern is the difference between the sidebearings of Text Bold in the Upright and Italic. In the Upright, the LSB is 12 and in the Italic, it is 42. 

Comma 

```
UPRIGHT:              ITALIC:
```

```
Text Regular          Text Regular
LSB: 12               LSB: 11
RSB: 71               RSB: 71
Width: 258            Width: 258
```

```
Text Regular [16]     Text Regular [16]
LSB: 51               LSB: 53
RSB: 72               RSB: 72
Width: 258            Width: 258
```

```
Text Bold             Text Bold
LSB: 12               LSB: 42
RSB: 52               RSB: 26
Width: 294            Width: 294
```

```
Text Bold [16]        Text Bold [16]
LSB: 54               LSB: 56
RSB: 60               RSB: 60
Width: 294            Width: 294
```

```
San Regular           San Italic
LSB: 8                LSB: -2
RSB: 55               RSB: 55
Width: 238            Width: 224
```

```
San Regular [16]      San Italic [16]
LSB: 51               LSB: 44
RSB: 52               RSB: 47
Width: 238            Width: 224
```

```
San Bold              San Bold Italic
LSB: 10               LSB: -2
RSB: 48               RSB: 48
Width: 288            Width: 271
```

```
San Bold [16]         San Bold Italic [16]
LSB: 50               LSB: 46
RSB: 56               RSB: 46
Width: 286            Width: 271

```

<img width="1280" alt="Screenshot 2020-05-14 at 14 35 48" src="https://user-images.githubusercontent.com/23149420/81983291-5e990e00-95f0-11ea-92fa-d1ba5cc60c03.png">

**Comment by chrissimpkins** [comment]
@Colophon-Foundry thoughts about this issue?

**Comment by Colophon-Foundry** [comment]
Good spot. The comma in the Bold Text Italic should be moved `-22`. (The semicolon is correct)

**Comment by chrissimpkins** [comment]
> Good spot. The comma in the Bold Text Italic should be moved `-22`. (The semicolon is correct)

@gorjious :+1: Good eye!

**Comment by chrissimpkins** [comment]
Added these changes for the next release

**Comment by chrissimpkins** [comment]
Freezing changes for v3.001 and pushing this back to v3.002.

**Comment by chrissimpkins** [comment]
This will be included in v3.004.  Gor is fixing this and will submit the changes as part of the Armenian script merge

**Comment by chrissimpkins** [comment]
> This will be included in v3.004. Gor is fixing this and will submit the changes as part of the Armenian script merge

Revised to v4.001 release

**Comment by chrissimpkins** [comment]
Merged in https://github.com/googlefonts/googlesans/pull/274 and will be released in v4.001
